tests/test_transformer.py

```diff
@@ -80,3 +191,17 @@ def test_from_wkt_bad_points(simplify_transformer):
 
     with pytest.raises(ShapelyError):
         simplify_transformer.from_wkt("")
+
+
+def test_from_geo_json_bad_points(simplify_transformer):
+    with pytest.raises(
+            Exception,
+            match=r"WKT: 'POINT \(30 10\)' is not a Polygon or MultiPolygon",
+    ):
+        simplify_transformer.from_geo_json({
+        "type": "Point",
```

Comment:
Im pretty sure the problem is that this shouldnt be at the same level. 
```suggestion
        simplify_transformer.from_geo_json({
            "type": "Point",
```

Reply:
Tragic. Didn't see that.